packages/flutter/test/material/tabs_test.dart

```diff
@@ -386,6 +386,54 @@ void main() {
       material3 ? const Size(14.25, 72.0) : const Size(14.0, 72.0));
   });
 
+  testWidgets('Material2 - Tab icon-label margin', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: false);
+    Widget buildApp() {
+      return MaterialApp(
+        theme: theme,
+        home: const Center(child: Material(
+          child: Tab(
+            icon: Icon(Icons.house),
+            text: 'x',
+          ),
+        ),
+      ));
+    }
+
+    double getIconMargin() {
+      final Rect iconRect = tester.getRect(find.byIcon(Icons.house));
+      final Rect labelRect = tester.getRect(find.text('x'));
+      return labelRect.top - iconRect.bottom;
+    }
+
+    await tester.pumpWidget(buildApp());
+    expect(getIconMargin(), 10);
+  });
+
+  testWidgets('Material3 - Tab icon-label margin', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData();
+    Widget buildApp() {
+      return MaterialApp(
+        theme: theme,
+        home: const Center(child: Material(
+          child: Tab(
+            icon: Icon(Icons.house),
+            text: 'x',
+          ),
+        ),
+      ));
+    }
+
+    double getIconMargin() {
+      final Rect iconRect = tester.getRect(find.byIcon(Icons.house));
+      final Rect labelRect = tester.getRect(find.text('x'));
+      return labelRect.top - iconRect.bottom;
+    }
+
+    await tester.pumpWidget(buildApp());
+    expect(getIconMargin(), 2);
```

Comment:
```suggestion
    expect(getIconMargin(), equals(2));
```

Making sure expect value is exactly `2`, not a double.

Reply:
@TahaTesser  Does `equals` have extra checks? I've always thought the `equals` matcher was implicit when passing a value. 
I'm trying `expect(10.0, equals(10))` and the expectation succeeds.
Which `expect` case are you referring it could fail?
`expect(10.0, 10)` also succeeds.

Reply:
It succeed with direct value as well. `equals`  structure of the expected value.

In this case, we're checking a  `double` value against an  `int`, whatever reason this changes from `10.00001` we still need make sure it is `10`.  This is not necessary but good to have.